tests/tests_nn/test_unet_2d.py

```diff
@@ -58,4 +65,11 @@ def test_unet_2d(shape, num_filters, num_pool_layers, skip, normalized):
 
     out = model(data, sens)
 
+    if openvino_available:
+        ov_model = OpenVINOModel(model)
+        ov_out = ov_model(masked_kspace=data, sensitivity_map=sens)
```

Comment:
Model launch should be equal: `ov_out = ov_model(data, sens)`

Reply:
Have to do a rebase, this test has been updated